### PR TITLE
Ses/health cvs r14

### PR DESCRIPTION
**Pull request by sebastianschramm**

### CODE REVIEW

**Comment by sebastianschramm** [comment]
I addded "has_metrics" choices to service_type. When you have time please review. Thanks :)

File: templates/health-r14-swagger/1/rancher-compose.yml [hunk 1]
```diff
@@ -27,7 +27,9 @@ version: '2'
       type: enum
       options:
         - webui
+        - webui, has_metrics
```
Comment (flypenguin) [inline templates/health-r14-swagger/1/rancher-compose.yml]: that just does not make sense. that thing either *has* metrics, or not. that should not be a choice, right? cause it's the software that is always the same, right? or do I not get something correctly here?

that you can choose the visibility *does* make sense though - sometimes you want to deploy internally for testing, sometimes externally for "production", etc.

so ... why?

**Comment by sebastianschramm** [comment]
I want to choose if prometheus picks up the metrics or not from my deployed
serrvice. E.g. I might want to deploy a test instance and not mess with the
metrics.

2017-08-09 11:09 GMT+02:00 Axel Bock <notifications@github.com>:

**Comment by pkaluzacog** [comment]
then please use a different form field, which defaults to the "has_metrics"
value. still not cool, cause you take metrics away from you which you might
need for testing.

and just as a side note (I don't know if you actually do this or not): each
service instance should have an identifier available to prometheus which
enables you to separate metrics from test instances away from the
production metrics, otherwise the whole setup is kinda useless :)



2017-08-09 11:15 GMT+02:00 Sebastian M. Schramm <notifications@github.com>:

**Comment by sebastianschramm** [comment]
ok, yes i can see "instance" and "job" information in grafana for each metric. hence having "has_metrics" always set is fine.
thanks for pointing this out :+1:

**Pull request closed without merging**